features/step_definitions/petition_steps.rb

```diff
@@ -65,7 +65,13 @@
   FactoryGirl.create(:open_petition, petition_attributes)
 end
 
-Given(/^a petition "(.*?)" exists and hasn't passed the threshold for a response$/) do |action|
+Given(/^a petition "([^"]*)" exists with a signature count of (\d+) and not set a debate date$/) do |title, count|
+  p = FactoryGirl.create(:open_petition)
+  p.update_attribute(:singature_count, count)
+  p.debate_outcome.debate_on == nil
+end
```

Comment:
I suspect this step isn't used because the attribute is actually ``debated_on`` so it would error.  Also, setting the a field to nil doesn't actually save the object, which is good because if it was the right field, setting debated_on to nil would make it invalid as debate outcomes need to have a debated_on date.  Probably just wanted to do ``p.debate_outcome = nil`` like in the other new steps (because, unlike setting attributes to ``nil``, setting a has_one to nil *does* save to the DB).  As it is I think we can just delete this step.